chapters/image_accceses.md

```diff
@@ -0,0 +1,146 @@
+# Image Accesses
+
+SPIR-V has various [Image Instructions](https://registry.khronos.org/SPIR-V/specs/unified1/SPIRV.html#_image_instructions) that interact with `OpTypeImage`. This chapter aims to give some extra context around these instructions.
+
+> There is an [example.frag](./examples/image_accesses/example.frag) (and [example.spv](./examples/image_accesses/example.spv)) GLSL shader that has an example of various ways images can be accessed in SPIR-V.
+
+# Coordinate Systems
+
+It is important to know there are 3 texel coordinate systems used for images:
+
+- Normalized (floats)
+- Unnormalized (floats)
+- Integers
+
+The SPIR-V spec documents which instructions are allowed to use which coordinate system.
+The [Vulkan Spec Texel Coordinate Systems Chapter](https://registry.khronos.org/vulkan/specs/1.3-extensions/html/vkspec.html#textures-texel-coordinate-systems) is a good resource to learn more about this.
+
+When using a `vec4` (4 wide vector) as a coordinate, the API spec defines which each vector element points to. As an example, in Vulkan, when using Unnormalized coordinates, the `vec4.z` element is the value of the `array layer`.
```

Comment:
Possible future upgrades here:

* I think there is a bug in "defines which each vector element", maybe "which" should be "what"?
* "points to" is potentially confusing, since the coordinates are not pointers.  Maybe "represents"?